### PR TITLE
fix_install_sm  fixing formatting in procedure for Installing and Con…

### PR DESCRIPTION
**Pull request by emarcusRH**
…figuring the RHV Manager /oVirt Engine

Fixes issue fix_install_sm

Changes proposed in this pull request:

- fixing interrupted numbering 
- fixing incorrect note indentation in the procedure (which caused the numbering problems)
- add "Next steps" title to final section of procedure
Install Guides /Installing and Configuring the RHV Manager/ oVirt Engine

Previews:
https://ovirt.github.io/ovirt-site/previews/2708/documentation/installing_ovirt_as_a_standalone_manager_with_remote_databases/index.html#Configuring_the_Red_Hat_Virtualization_Manager_install_RHVM
see steps 4 and 13, as well as following step 22 - "Next steps"

https://ovirt.github.io/ovirt-site/previews/2708/documentation/installing_ovirt_as_a_standalone_manager_with_local_databases/index.html#Configuring_the_Red_Hat_Virtualization_Manager_install_RHVM
see step 4, and following step 24 - "Next steps"

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @emarcusRH )

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

File: source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc [hunk 1]
```diff
@@ -281,7 +281,7 @@ Confirm engine admin password:
 ----
 Application mode (Both, Virt, Gluster) [Both]:
 ----
-*Both* offers the greatest flexibility. In most cases, select `Both`. *Virt* allows you to run virtual machines in the environment; *Gluster* only allows you to manage GlusterFS from the Administration Portal.
+*Both* - offers the greatest flexibility. In most cases, select `Both`. *Virt* - allows you to run virtual machines in the environment; *Gluster* - only allows you to manage GlusterFS from the Administration Portal.
```
Comment (RichardHoch) [inline source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc:284]: 1. I don't think hyphen are the best choice. Consider one of the following possibilities:
a. no punctuation. The sentences read clearly without it.
b: a colon
c: rewrite as a bulleted list
2. re: "In most cases, select `Both`. I would *Bold* ""Both," unless the user actually types it -- in that case, i'd use "type" as the verb, not select.

File: source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc [hunk 2]
```diff
@@ -351,9 +351,12 @@ endif::SM_localDB_deploy[]
...
 
-. Install the certificate authority according to the instructions provided by your browser. You can get the certificate authority's certificate by navigating to `http://_manager-fqdn_/ovirt-engine/services/pki-resource?resource=ca-certificate&amp;format=X509-PEM-CA`, replacing _manager-fqdn_ with the FQDN that you provided during the installation.
+* Install the certificate authority according to the instructions provided by your browser. You can get the certificate authority's certificate by navigating to `http://_manager-fqdn_/ovirt-engine/services/pki-resource?resource=ca-certificate&amp;format=X509-PEM-CA`, replacing _manager-fqdn_ with the FQDN that you provided during the installation.
```
Comment (RichardHoch) [inline source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc:362]: I know it's legacy text but the link looks odd in the preview and it contains italics:  "_manager-fqdn_", which is unusual for a link. Please double-check this.

**Comment by RichardHoch** [comment]
General note: The Admonitions ("NOTE," etc.) aren't rendering correctly in the preview and "Next steps" readers as plain text.  can anything be done about that? (You might use a Heading 2 for Next steps). 

A couple small notes, but otherwise LGTM.

**Review by RichardHoch — APPROVED** [review]

File: source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc (hunk 1 above)
Comment (emarcusRH) [reply source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc:284]: I found the wording slightly annoying as is - "Both offers the..." looked like a typo at first, so I added the hyphens to make it unambiguous.  Used your suggestion for bulleted list...

**Comment by emarcusRH** [comment]
> General note: The Admonitions ("NOTE," etc.) aren't rendering correctly in the preview and "Next steps" readers as plain text. can anything be done about that? (You might use a Heading 2 for Next steps).
> 
> A couple small notes, but otherwise LGTM.
Yes, the oVirt preview is not formatted as nicely as the RHV  doc portal...
https://jenkins.dxp.redhat.com/job/CCS/job/ccs-mr-preview/17717/artifact/assembly-Installing_Red_Hat_Virtualization_as_a_standalone_Manager_with_remote_databases/preview/index.html#Configuring_the_Red_Hat_Virtualization_Manager_install_RHVM

**Pull request merged**